docs/docs/guides/troubleshooting.md

```diff
@@ -29,18 +31,18 @@ wet-mangust-1 provisioning completed (failed)
 All provisioning attempts failed. This is likely due to cloud providers not having enough capacity. Check CLI and server logs for more details.
 ```
 
-#### Backend configuration
+#### Cause 1: Backend configuration
```

Comment:
Not a fan of repeating Cause # multiple times - make reading the document a bit hard

Reply:
It actually looks much better than current headings when rendered. Because currently it's hard to visually differentiate between problems and possible causes/solutions:

<img width="764" alt="Screenshot 2024-12-25 at 10 43 56" src="https://github.com/user-attachments/assets/b5bb45c2-0b2a-4bb4-9dbd-7291a4fe27f3" />